assembl/static2/js/app/components/debate/thread/post.jsx

```diff
@@ -301,17 +300,17 @@ export class EmptyPost extends React.PureComponent {
             </div>
           </div>
         </div>
-        {this.state.showAnswerForm ? (
-          <div className="answer-form">
-            <AnswerForm
-              parentId={id}
-              ideaId={ideaId}
-              refetchIdea={refetchIdea}
-              textareaRef={answerTextareaRef}
-              hideAnswerForm={this.hideAnswerForm}
-            />
-          </div>
-        ) : null}
+        <div className={this.state.showAnswerForm ? 'answer-form' : 'collapsed-answer-form'}>
+          <AnswerForm
+            parentId={id}
+            ideaId={ideaId}
+            refetchIdea={refetchIdea}
+            textareaRef={answerTextareaRef}
+            hideAnswerForm={this.hideAnswerForm}
```

Comment:
are you sure `hideAnswerForm` is still used ?

Reply:
Yes It's still used